docs/modules/plugins/pages/plugin-browser-stack.adoc

```diff
@@ -46,3 +46,45 @@ Explore https://www.browserstack.com/app-automate/capabilities?tag=w3c[Capabilit
 * See xref:tests-development:tests-configuration.adoc#_selenium_grid[Selenium Grid] properties to configure generic options.
 * Browser Stack specific properties must be prepended with `selenium.grid.capabilities.bstack\:options.` prefix, meaning given a property `projectName` with a value `My Project` the resulting property will be `selenium.grid.capabilities.bstack\:options.projectName=My Project`
 * Appium specific properties in Browser Stack must be prepended with `selenium.grid.capabilities.appium\:options.` prefix, meaning given a property `clearSystemFiles` with a value `true` the resulting property will be `selenium.grid.capabilities.appium\:options.clearSystemFiles=true`
+
+== Mobile Steps
+
+=== Save network logs
+
+==== *_Info_*
```

Comment:
please follow structure similar to this one: https://docs.vividus.dev/vividus/0.2.11-SNAPSHOT/plugins/plugin-rest-api.html#_init_multipart_http_request